packages/klighd-core/src/options/render-options-registry.ts

```diff
@@ -61,8 +80,35 @@ export class ShowConstraintOption implements RenderOption {
     readonly name: string = ShowConstraintOption.NAME;
     readonly type: TransformationOptionType = TransformationOptionType.CHECK;
     readonly initialValue: boolean = false;
+    readonly renderCategory: string = Appearance.ID
+    readonly description = 'Show marker for nodes that have layout constraints set.'
```

Comment:
…that have *interactive* layout constraints set

Reply:
ok